legal/security_research.md

```diff
@@ -0,0 +1,60 @@
+
+- Avoid explicit use of an individual's name or their precise words (paraphrase instead)
+- Avoid implications of a particular individual's direct culpability in a particular issue (focus on systematics instead)
+- Avoid attributing insufficiencies to lack of care or attention
+- Where applicable provide suggestions for implementable improvements
+
+The goal of these guidelines is to encourage researchers to engage in meaningful dialog with the community. In many cases, process suboptimalities are known, but hard to address for lack of available volunteer capacity, funding or a combination thereof. Researchers can help by providing concrete data to understand the scope and extent of a particular problem and by providing concrete suggestions.
+
+## Interventional research
+
+Stricter requirements apply for research that involves direct interaction of the researchers or their agents and the community (for purposes other than ordinary collaboration on software development). Such research is considered *human subject research* should be handled carefully.
+
+Such research may include (but is not limited to):
+
+- Developer Surveys
+- The posting of messages/issues/PRs to ascertain community responsiveness
+- The use of multiple artificial accounts/profiles/personas
+- Attempts at the deliberate introduction of non-functional or malicious code
+- Automated interaction with the community (e.g. the automated filing of GitHub issues, Discourse posts, etc.)
+
+It is the Julia project's policy to allow such research provided that each interaction includes a clear statement of:
+
+1. The fact that is it part of a research project
+2. The identity of the research project's PI
+3. The goal and methodologies of the research project (may be provided as a link)
+4. Information on how to request cessation of activities, deletion of specific data or a general opt-out
+
+If such disclosure is incompatible with the objectives of your research, strict procedures apply.
+In such cases, please email the [Julia Stewards](/community/stewards/) at [stewards@julialang.org](mailto:stewards@julialang.org) with a description of the following:
+
+- The goals of your research
+- The proposed methodologies of your research
+- The identity of the project PI and all project staff
+- The source (agencies, contract numbers, program managers) of all funding for your research
+- The handles of all user accounts that will be used to interact with the community in furtherance of the research
+- Any IRB review opinions of your research methodology (N.B.: for this kind of research, IRB approval will likely be a condition of approval by Stewards)
+
+**DO NOT** begin your research unless and until you have received approval from Stewards. If you are contemplating such research, you are encouraged to contact Stewards early to discuss your plans.
```

Comment:
Worth mentioning that this potentially shifts the role of the stewards from an internal dispute resolution group to one which acts on behalf of the Julia users with respect to these sort of research studies.